utils/storage/beaconQueue.js

```diff
@@ -96,7 +96,7 @@ class BeaconQueue {
     });
     const data = { batch };
     const payload = JSON.stringify(data, this.replacer);
-    const blob = new Blob([payload], { type: "application/json" });
+    const blob = new Blob([payload], { type: "text/plain" });
```

Comment:
Can you please address the following?
- Results before and after this change
- If not already, please double-check the changes with the server team.